FV3/coarse_graining/coarse_graining.F90

```diff
@@ -75,17 +79,24 @@ module coarse_graining_mod
      module procedure weighted_block_edge_average_y_pre_downsampled_masked
   end interface weighted_block_edge_average_y_pre_downsampled
 
+  interface vertical_remapping_requirements
+      module procedure vertical_remapping_requirements_pressure_level
+      module procedure vertical_remapping_requirements_blended_area_weighted
+  end interface
+
   ! Global variables for the module, initialized in coarse_graining_init
   integer :: is, ie, js, je, npz
   integer :: is_coarse, ie_coarse, js_coarse, je_coarse
   character(len=11) :: MODEL_LEVEL = 'model_level'
   character(len=14) :: PRESSURE_LEVEL = 'pressure_level'
+  character(len=21) :: BLENDED_AREA_WEIGHTED = 'blended_area_weighted'
+  real :: sigma_blend = 0.9  ! Constant defining sigma level at which we switch to pressure-level coarsening in the blended method.
```

Comment:
Just double checking that we don't envision changing this value? Or is this something that can be configured in the namelist?